lib/connectors/kv-memory.js

```diff
@@ -118,6 +122,28 @@ function(modelName, key, ttl, options, callback) {
   process.nextTick(callback);
 };
 
+KeyValueMemoryConnector.prototype.ttl =
+function(modelName, key, options, callback) {
+  this._removeIfExpired(modelName, key);
+
+  var store = this._getStoreForModel(modelName);
+  var error = new Error('Cannot get TTL for unknown key ' + key);
```

Comment:
I find this strange, why are you creating `error` even on the happy path when it's not used?